admin/google-blockly/own/blocks_words.js

```diff
@@ -115,47 +115,47 @@ Blockly.Words['toggle_ms']                            = {'en': 'in ms',
 Blockly.Words['get_value_help']                       = {'en': 'getstate',                                        'de': 'getstate',                                        'ru': 'getstate',                                        'pt': 'getstate',                                        'nl': 'getstate',                                        'fr': 'getstate',                                        'it': 'GetState',                                        'es': 'Gettate',                                         'pl': 'getstate',                                        'zh-cn': '获取对象的指定属性值'};
 Blockly.Words['get_value_default']                    = {'en': 'select ID',                                       'de': 'ID auswählen',                                    'ru': 'Выбрать ID',                                      'pt': 'selecione ID',                                    'nl': 'selecteer ID',                                    'fr': 'sélectionnez ID',                                 'it': 'seleziona ID',                                    'es': 'seleccionar ID',                                  'pl': 'select ID',                                       'zh-cn': '对象ID'};
 Blockly.Words['get_value_val']                        = {'en': 'Value',                                           'de': 'Wert',                                            'ru': 'Значение',                                        'pt': 'Valor',                                           'nl': 'Waarde',                                          'fr': 'Valeur',                                          'it': 'Valore',                                          'es': 'Valor',                                           'pl': 'Value',                                           'zh-cn': '值'};
-Blockly.Words['get_value_ack']                        = {'en': 'Acknowledge',                                     'de': 'anerkannt',                                       'ru': 'Подтверждение',                                   'pt': 'confirmado',                                      'nl': 'Erkennen',                                        'fr': 'Reconnaître',                                     'it': 'Riconoscere',                                     'es': 'Reconocer',                                       'pl': 'Acknowledge',                                     'zh-cn': '是否应答'};
+Blockly.Words['get_value_ack']                        = {'en': 'Acknowledge',                                     'de': 'Anerkannt',                                       'ru': 'Подтверждение',                                   'pt': 'confirmado',                                      'nl': 'Erkennen',                                        'fr': 'Reconnaître',                                     'it': 'Riconoscere',                                     'es': 'Reconocer',                                       'pl': 'Acknowledge',                                     'zh-cn': '是否应答'};
```

Comment:
"Bestätigt" would be better (since that word is also used in the admin adapter)

Reply:
Yes, you're right. I changed it.